src/index.spec.ts

```diff
@@ -1188,24 +1187,24 @@ describe('EppoClient config', () => {
         json: () => Promise.resolve({}),
       });
     }) as jest.Mock;
-    EppoJSClient.initialized = false;
     const client = await init({
+      forceReinitialize: true,
```

Comment:
The "official" way to ensure a new instance is spun up (vs. flipping the boolean)